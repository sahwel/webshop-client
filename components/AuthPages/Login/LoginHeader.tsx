import React from 'react';

const LoginHeader = () => {
  return (
    <div className="select-none">
      <h1 className="text-[300%]">Let’s sign you in!</h1>
      <h3>Welcome back,</h3>
      <h3>you’ve been missed!</h3>
    </div>
  );
};

export default LoginHeader;
