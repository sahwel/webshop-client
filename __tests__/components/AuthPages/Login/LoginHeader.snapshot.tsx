import { render } from '@testing-library/react';
import LoginHeader from '@/components/AuthPages/Login/LoginHeader';

describe('LoginHeader snapshot', () => {
  it('renders LoginHeader unchanged', () => {
    const { container } = render(<LoginHeader />);
    expect(container).toMatchSnapshot();
  });
});
