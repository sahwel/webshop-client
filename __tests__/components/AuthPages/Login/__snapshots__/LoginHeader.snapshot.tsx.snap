// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginHeader snapshot renders LoginHeader unchanged 1`] = `
<div>
  <div
    class="select-none"
  >
    <h1
      class="text-[300%]"
    >
      Let’s sign you in!
    </h1>
    <h3>
      Welcome back,
    </h3>
    <h3>
      you’ve been missed!
    </h3>
  </div>
</div>
`;
